src/uwtools/tests/api/test_drivers.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-function-docstring,protected-access
+
+from datetime import datetime as dt
+from unittest.mock import patch
+
+import iotaa
+import pytest
+
+from uwtools.api import (
```

Comment:
This definitely makes the duplication of code much less and I appreciate in the future it'll feel like a real addition by adding a driver vs. a copy-paste job.